Source/WebCore/rendering/RenderSearchField.h

```diff
@@ -44,6 +44,7 @@ class RenderSearchField final : public RenderTextControlSingleLine, private Popu
     bool popupIsVisible() const { return m_searchPopupIsVisible; }
     void showPopup();
     void hidePopup();
+    Vector<RecentSearch> recentSearches() const { return m_recentSearches; }
```

Comment:
This is unnecessarily inefficient because it copies the vector every time it’s called. Could return a `Span` since the caller just needs to iterate, not copy.

I guess it’s just for testing but even so I would prefer a more efficient idiom.

Reply:
Sure, I will change.